tests/src/tools/ReadyToRun.TestHarness/PathHelpers.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.IO;
+
+namespace ReadyToRun.TestHarness
+{
+    /// <summary>
+    /// A set of helper to manipulate paths into a canonicalized form to ensure user-provided paths
+    /// match those in the ETW log.
+    /// </summary>
+    static class PathHelpers
```

Comment:
Nit - I was under the impressions that we typically call such classes "###Extensions", not 100% sure how much of a code style requirement that is.